jobs/golangapiserver/spec

```diff
@@ -222,6 +222,13 @@ properties:
   autoscaler.changeloglock_timeout_seconds:
     default: 180
     description: "Liquibase changelog lock timeout duration in seconds"
+
+  autoscaler.apiserver.rate_limit.valid_duration:
+    description: "Accept at most 10(max_amount) requests every 1s(valid_duration)"
```

Comment:
Seem this is not a description. @cdlliuy your idea?

Reply:
The rate limit evaluation duration in seconds